spec/factories/squad_factory.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Squadster.SquadFactory do
+  defmacro __using__(opts) do
+    quote do
+      def squad_factory do
+        %Squadster.Formations.Squad{
+          squad_number: "squad #{Faker.random_between(1, 1000)}",
```

Comment:
Not a big deal, but it's better to use value from seeds, it should be 6 numbers. Or why are we need "squad" prefix?